GitHubComments.ps1

```diff
@@ -0,0 +1,576 @@
+        'AcceptHeader' = (Get-CommentAcceptHeader -MediaType $MediaType -MediaTypeVersion $MediaTypeVersion)
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+    }
+
+    return Invoke-GHRestMethod @params
+}
+
+function Set-GitHubComment
+{
+<#
+    .DESCRIPTION
+        Set an existing comment in an issue for the given repository
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER CommentID
+        The comment ID of the comment to edit.
+
+    .PARAMETER body
+        The new contents of the comment.
+
+    .PARAMETER MediaType
+        The format in which the API will return the body of the comment.
+
+        raw - Return the raw markdown body. Response will include body. This is the default if you do not pass any specific media type.
+        text - Return a text only representation of the markdown body. Response will include body_text.
+        html - Return HTML rendered from the body's markdown. Response will include body_html.
+        full - Return raw, text and HTML representations. Response will include body, body_text, and body_html.
+
+    .PARAMETER MediaTypeVersion
+        The version of the response to return. The default version of the API may change in the future.
+        If you're building an application and care about the stability of the API, be sure to request 
+        a specific version. More info at https://developer.github.com/v3/media/.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Set-GitHubComment -OwnerName Powershell -RepositoryName PowerShellForGitHub -CommentID 1 -Body "Testing this API"
+
+        Set an existing comment in an issue for the PowerShell\PowerShellForGitHub project.
```

Comment:
I'd rephrase this as "Update an existing...."  I think there's a different between how PowerShell users name/think of methods, and how we'd _describe_ those methods.